src/collaborative/revisions.ts

```diff
@@ -19,12 +21,14 @@ export class Revision implements RevisionData {
     id: UID,
     clientId: ClientId,
     commands: readonly CoreCommand[],
-    changes?: readonly HistoryChange[]
+    changes?: readonly HistoryChange[],
+    timestamp?: number
```

Comment:
What do you think of a default value?
```suggestion
    timestamp?: number = Date.now()
```